actors/builtin/miner/policy.go

```diff
@@ -297,8 +297,8 @@ func RewardForDisputedWindowPoSt(proofType abi.RegisteredPoStProof, disputedPowe
 }
 
 const MaxAggregatedSectors = 819
-const MinAggregatedSectors = 1
-const MaxAggregateProofSize = 192000
+const MinAggregatedSectors = 4
+const MaxAggregateProofSize = 81960
```

Comment:
Note to (likely) self -> write some higher-level test in lotus which posts maximum-size proof to the chain to make sure we don't have any length checks which could be too tight

Reply:
Great idea. FYI I made this 2x the size of the expected largest proof here to try to prevent problems